demisto_sdk/tests/test_files/invalid_py3_XSOARLinter.py

```diff
@@ -857,7 +858,7 @@ def split_fields(fields: str = '') -> dict:
     Returns:
         dic_fields object for request.
     """
-    print('tests')
+    logger.info('tests')
```

Comment:
please undo - this is a test file

Reply:
https://github.com/demisto/demisto-sdk/pull/2917/files#diff-3602faaca6fa8fe18eb2820af2c85f6e2463d5d695040386db3221ad99917746